src/components/SidebarItems.tsx

```diff
@@ -65,13 +64,11 @@ const SidebarItems: FC<{ cat: union[number]; active: boolean }> = ({
   return (
     <div
       className={`${
-        (active || mouseOver) && cat != 'Events'
-          ? '-my-2.5 mb-2.5 rounded-3xl bg-white py-2.5 shadow-md'
-          : (active || mouseOver) && cat == 'Events'
-          ? '-my-2.5 rounded-3xl bg-white py-2.5 shadow-md'
-          : cat != 'Events'
-          ? 'mb-5'
-          : ''
+        active || mouseOver
+          ? `-my-2.5 rounded-3xl bg-white py-2.5 shadow-md ${
+              cat == 'Events' ? '' : 'mb-2.5'
```

Comment:
tailwind has a [last](https://tailwindcss.com/docs/hover-focus-and-other-states#first-last-odd-and-even) modifier that should make it work regardless of which item is the last one and wouldn't need a ternary. may need to be something like "mb-2.5 last:-mb-2.5"

Reply:
Thanks for the suggestion. That definitely made it a lot simpler.